src/main/scala/org/apache/spark/sql/matfast/partitioner/ColumnPartitioner.scala

```diff
@@ -21,7 +21,7 @@ import org.apache.spark.{Partitioner, SparkConf}
 import org.apache.spark.rdd.{RDD, ShuffledRDD}
 import org.apache.spark.sql.catalyst.InternalRow
 import org.apache.spark.sql.matfast.util.MatfastSerializer
-
+// scalastyle:off
```

Comment:
If we just add // scalastyle:off without tuning it on, then all the code below will not go through the style checking, right? Why do we skip the style checking?